lib/rules/__tests__/index.test.js

```diff
@@ -58,3 +59,45 @@ describe('custom message option', () => {
 		},
 	);
 });
+
+describe('recommended and standard configs', () => {
+	const cwd = path.join(__dirname, 'tmp');
+
+	fs.rmSync(cwd, { recursive: true, force: true });
+	fs.mkdirSync(cwd, { recursive: true });
+	fs.writeFileSync(path.join(cwd, 'package.json'), '{}');
+	cp.execSync(
+		'npm install --silent --no-package-lock --no-audit --omit=peer stylelint-config-recommended stylelint-config-standard',
```

Comment:
[note] For details about options, see https://docs.npmjs.com/cli/v9/commands/npm-install